data/test/tabletserver/exec_cases.txt

```diff
@@ -1075,6 +1075,41 @@
   "FullQuery": "delete from c"
 }
 
+# sequence
+"select next value for seq from dual"
```

Comment:
I don't think we need the 'from dual' part here, do we? The from clause is optional? (pardon my limited SQL knowledge :) )